packages/sourcemap/src/index.ts

```diff
@@ -67,11 +69,11 @@ export async function indexJsMapFileAsync(organization: string, hashAlgo: string
     }
 
     // Update the sourcemap file.
-    console.log(`    Updating sourcemap file with key ${clientKey}`);
+    log(`    Updating sourcemap file with key ${clientKey}`);
     setClientKeyOnSourceMap(clientKey, sourceMap)
     await fsExtra.writeJSON(jsMapFile, sourceMap);
 
-    console.log(`    Updating source file ${sourceFilePath}`);
+    log(`    Updating source file ${sourceFilePath}`);    
```

Comment:
Removing extra trailing  whitespace
```suggestion
    log(`    Updating source file ${sourceFilePath}`);
```